mappings/net/minecraft/world/gen/decorator/CountMultiplayerDecorator.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_5452 net/minecraft/world/gen/decorator/CountMultiplayerDecorator
```

Comment:
It's "multilayer", not "multiplayer"

Reply:
Can you tell I was running on very little sleep when mapping? Haha, will fix.